geoips/interfaces/module_based/output_checkers.py

```diff
@@ -145,7 +145,9 @@ def get_out_diff_fname(self, compare_product, output_product, ext=None, flag=Non
             out_diff_fname = splitext(out_diff_fname)[0] + ".png"
         return out_diff_fname
 
-    def compare_outputs(self, compare_path, output_products):
+    def compare_outputs(
+        self, compare_path, output_products, image_compare_threshold="medium"
+    ):
```

Comment:
If we created "output_checker_kwargs" command line argument, then we could just pass it in as {"image_compare_threshold": "medium"}, and it would also allow us to potentially pass in other kwargs as needed?

Reply:
Also, if this is an overall argument for ALL output checkers (since it is passed into the "compare_outputs" method), should it be named something other than "image_compare_threshold", since there are arbitrary output formats besides just images ?

Maybe if we went the "output_checker_kwargs" option, then we just pass "output_checker_kwargs" into "compare_outputs", then they get passed into the "outputs_match" method appropriately?

Reply:
Done!